app/views/seeds/index.rss.haml

```diff
@@ -16,8 +16,9 @@
             <p>GMO? #{seed.gmo}</p>
             <p>Heirloom? #{seed.heirloom}</p>
           - if seed.tradable?
-            :escaped
-              <p>Will trade #{seed.tradable_to} from #{seed.owner.location ? seed.owner.location : 'unknown location'}</p>
+            %p
```

Comment:
Where's the escaping happening now?

Our location fields appear to be [completely unclean](http://www.staging.growstuff.org/members/pozorvlak). Oops.

```
irb(main):002:0> me = Member.find('pozorvlak')
  Member Load (2.3ms)  SELECT  "members".* FROM "members" WHERE "members"."slug" = $1  ORDER BY lower(login_name) asc LIMIT 1  [["slug", "pozorvlak"]]
  Member Load (2.3ms)  SELECT  "members".* FROM "members" WHERE "members"."slug" = $1  ORDER BY lower(login_name) asc LIMIT 1  [["slug", "pozorvlak"]]
irb(main):003:0> me.location
=> "<script>alert('Hello, world!')</script>"
```